pkg/quick/quick_test.go

```diff
@@ -130,38 +154,63 @@ func (s *MySuite) TestLoadFile(c *C) {
 	}
 	saveMe := myStruct{}
 	_, err := Load("test.json", &saveMe)
-	c.Assert(err, Not(IsNil))
+	if err == nil {
```

Comment:
There is no error to log. Should be changed to a text, more like, Unexpected, should have failed while loading test.json.

Reply:
Fixed.